lib/build/builder.ts

```diff
@@ -278,7 +272,7 @@ function getDockerDaemonBuildOutputParserStream(
 			try {
 				if (data.error) {
 					throw new Error(data.error);
-				} else {
+				} else if (data.stream != null) {
```

Comment:
This is necessary because with native streams writing null/undefined is an error, it also just avoids unnecessary work when nothing is going to match against it anyway